tools/payments/cmd/chain-account/main.go

```diff
@@ -0,0 +1,238 @@
+/*
+chain-account allows payment operators to add addresses to be used for on-chain payouts.
+
+Usage:
+
+chain-account [flags] generatePUBLIC_KEY
+
+The flags are:
+
+	-pr
+		Location on file system of the original prepared report
+	-v
+		verbose logging enabled
+	-k
+		Location on file system of the operators private ED25519 signing key in PEM format.
+	-e
+		The environment to which the operator is sending approval for transactions.
+*/
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/ed25519"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	client "github.com/brave-intl/bat-go/libs/clients"
+	"github.com/brave-intl/bat-go/libs/httpsignature"
+	"github.com/brave-intl/bat-go/libs/payments"
+	paymentscli "github.com/brave-intl/bat-go/tools/payments"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// command line flags
+	operatorKey := flag.String(
+		"k", "test/private.pem",
+		"the operator's key file location (ed25519 private key) in PEM format")
+
+	env := flag.String(
+		"e", "local",
+		"the environment to which the tool will interact")
+
+	verbose := flag.Bool(
+		"v", false,
+		"view verbose logging")
+
+	pcr2 := flag.String(
+		"pcr2", "", "the hex PCR2 value for this enclave")
+
+	flag.Parse()
+
+	args := flag.Args()
+	if len(args) < 2 {
+		log.Fatal("Expected chain id and action arguments")
+	}
+
+	switch args[0] {
+	case "solana":
+		switch args[1] {
+		case "generate":
+			generateSolanaAddress(ctx, *operatorKey, *env, *pcr2, *verbose)
+		case "approve":
+			if len(args) < 3 {
+				log.Fatal("Expected public key argument for approval")
+			}
+			pubKey := args[2]
+			approveSolanaAddress(ctx, pubKey, *operatorKey, *env, *pcr2, *verbose)
+		default:
+			log.Fatal("unrecognized solana command")
+		}
+	default:
+		log.Fatal("unrecognized chain id")
+	}
+
+}
+
+func generateSolanaAddress(ctx context.Context, key, env, pcr2 string, verbose bool) {
+	if env != "development" && len(pcr2) != 96 {
```

Comment:
it probably makes sense to drop this all together later to make the dev environment match. If there's reason to keep this for now then no issues with leaving it in.